tests/server-integration/SetField/SetFieldTest.hs

```diff
@@ -158,3 +116,51 @@ unit_set_field_updates_modification_date = cofferTest do
 
   assertBool "Dates are not equal"
     $ responseBody response ^?! key "fields" . key "field" . key "dateModified" . _JSON == newModifiedDate
+
+unit_set_field_visibility :: IO ()
+unit_set_field_visibility = cofferTest do
+  createEntry "dir/entry"
+  void $ setField "dir/entry" "private-field" Nothing "contents"
+  void $ setField "dir/entry" "public-field" Nothing "contents"
+  changeFieldVisibility "dir/entry" "private-field" "private"
+  changeFieldVisibility "dir/entry" "public-field" "public"
```

Comment:
Why there is two checks in one test?

If we're expecting to fail at once = one way (only private/public)
If it could fail on one, but not on second variant seems as two separate tests

I know that they're just doing the same thing, but we aren't short on test space)

Reply:
Fixed :+1: